.github/workflows/template_automerge_dependabot.yml

```diff
@@ -44,11 +48,30 @@ jobs:
         run: |
           gh pr review --approve "$PR_URL"
 
-          if [ ${{ inputs.force }} == 'true' ]; then
-            gh pr merge "$PR_URL" --squash --admin
+          MERGE_OPTIONS=()
+
+          case "${{ env.STRATEGY }}" in
+            "rebase")
+              MERGE_OPTIONS+=("--rebase")
+              ;;
+            "merge")
+              MERGE_OPTIONS+=("--merge")
+              ;;
+            *)
+              MERGE_OPTIONS+=("--squash")
+              ;;
+          esac
+
+          if [ "${{ env.FORCE }}" == 'true' ]; then
+            MERGE_OPTIONS+=("--admin")
           else
-            gh pr merge --auto --merge "$PR_URL"
+            MERGE_OPTIONS+=("--auto")
           fi
+
+          echo "Executing merge command with the options: '${MERGE_OPTIONS[*]}'"
+          gh pr merge "$PR_URL" "${MERGE_OPTIONS[@]}"
         env:
           PR_URL: ${{ github.event.pull_request.html_url }}
           GITHUB_TOKEN: ${{ steps.get_token.outputs.token }}
+          STRATEGY: ${{ inputs.strategy }}
```

Comment:
What is the advantage to have it in the env vars? 🤔

Reply:
There was a misunderstanding which Falk an I clarified in a call. 👍